hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationInterceptorREST.java

```diff
@@ -2117,7 +2128,7 @@ private <R> Map<SubClusterInfo, R> invokeConcurrent(Collection<SubClusterInfo> c
         throw new YarnRuntimeException(e.getCause().getMessage(), e);
       }
     });
-
+    
```

Comment:
You have a warning here.

Reply:
Thank you very much for helping to review the code, I will fix it.